actions/delegator/secure-attestations-download/action.yml

```diff
@@ -0,0 +1,23 @@
+name: "Secure attestations download"
+description: "Download a set of attestations and verify their SHA256"
```

Comment:
is this path acceptable for a public Action? Will "delegator" be confusing to end-users?